android/app/src/main/java/com/mattermost/share/ShareModule.java

```diff
@@ -75,7 +75,10 @@ public void clear() {
     @ReactMethod
     public void close(ReadableMap data) {
         this.clear();
-        getCurrentActivity().finish();
+        Activity currentActivity = getCurrentActivity();
+        if (currentActivity != null) {
```

Comment:
what is the side effect of not calling finish, and why would the currentActivity be null?

Reply:
ok lets go with this and we can see what else to do later on if needed